tests/tests.js

```diff
@@ -193,13 +183,15 @@ QUnit.test("generateEmptyCalendar test", function( assert ) {
         assert.equal($(this).html(), daysOfWeek[index]);
     });
     
-    //TODO test with a calendar with more than one month to make sure
-    // the second month does not get a yearHeader.
-    
+    this.checkit.addMonth(this.calendar);
+    $calendarDiv.empty();
```

Comment:
If you need to do this, then you probably should start a second test and give them better names.